fairlearn/_input_validation.py

```diff
@@ -10,6 +10,7 @@
 logger = logging.getLogger(__file__)
 
 _KW_SENSITIVE_FEATURES = "sensitive_features"
+_KW_CONTROL_FEATURES = "control_features"
```

Comment:
I don't think that we need these two constants. We are very strongly locked to "sensitive_features" and "control_features" and using `_KW_*` makes things look complicated.

Reply:
I'm not committed to the `_KW` prefix, but in general, it's better to keep things like this as separate constants.

Reply:
why?

Reply:
I'm fine keeping as is for now. But this was my reasoning:
* we use the optional argument `sensitive_features` all over the code and documentation, so if we decide to change that convention, we need to do some form of extensive Find/Replace anyway.
* having a constant called `_KW_SENSITIVE_FEATURES` slightly simplifies Find/Replace exercise, but introduces the need to include `import _KW_SENSITIVE_FEATURES` whenever I want to use `'sensitive_features'` string. That's a rather non-trivial dependency. It also seemingly results in a harder to read code. Do we want to have a master file `fairlearn/_constants.py` with all of keyword attributes that we use?
* at most places we should use explicit keyword arguments, so the use case of `_KW_SENSITIVE_FEATURES` is very limited (we might be able to remove it altogether).

Reply:
After the latest revisions, `_KW_SENSITIVE_FEATURES` is only use in `_input_validation.py` and `_KW_CONTROL_FEATURES` only pops up when `ThresholdOptimizer` says "can't to that."